packages/@aws-cdk/aws-ecs-patterns/test/fargate/queue-processing-fargate-service.test.ts

```diff
@@ -350,6 +350,7 @@ testDeprecated('test Fargate queue worker service construct - with optional prop
   new ecsPatterns.QueueProcessingFargateService(stack, 'Service', {
     cluster,
     memoryLimitMiB: 512,
+    ephemeralStorageGiB: 100,
```

Comment:
Same as above: Let's add in validation failure cases for below 21 and above 200.